docs/includes/enable-debugging-on-edge-devtools.md

```diff
@@ -0,0 +1,38 @@
+When the add-in is running in Microsoft Edge, UI-less code will not be able to attach to a debugger by default.
+UI-less code is any code running while the task pane is not visible, such as add-in commands. To enable debugging, you need to run the following [Windows PowerShell](https://docs.microsoft.com/powershell/scripting/getting-started/getting-started-with-windows-powershell) commands.
```

Comment:
"To enable debugging, run the following...."
(Per updates in the Microsoft Voice guidelines, we are trying to kill off all the "you cans" and "there is/are/weres"...)